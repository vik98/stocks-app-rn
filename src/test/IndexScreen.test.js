import React from "react";
import renderer from "react-test-renderer";
import IndexScreen from "../screens/IndexScreen";
describe("<IndexScreen/>", () => {
  it("has 1 child", () => {
    const tree = renderer.create(<IndexScreen />).toJSON();
    expect(tree.children.length).toBe(1);
  });
  it("renders correctly across screens", () => {
    const tree = renderer.create().toJSON();
    expect(tree).toMatchSnapshot();
  });
});
