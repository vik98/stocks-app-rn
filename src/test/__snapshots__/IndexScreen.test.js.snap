// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IndexScreen/> has 1 child 1`] = `1`;

exports[`<IndexScreen/> renders correctly across screens 1`] = `null`;
